templates/web/base/report/display.html

```diff
@@ -45,6 +45,8 @@
 [% END %]
 
   </div>
+
+  [% TRY %][% PROCESS second_column %][% CATCH file %][% END %]
```

Comment:
Just to avoid the ugly TRY, you could have `[% second_column %]` here, and `[% second_column = BLOCK %]` in the other file.